vulnfeeds/pypi/pypi.go

```diff
@@ -131,8 +131,15 @@ func hasPrefix(list []string, item string) bool {
 func processMatches(names []string) []string {
 	// Normalize all PyPI package names.
 	normalized := make([]string, 0, len(names))
+	encountered := map[string]bool{}
 	for _, name := range names {
-		normalized = append(normalized, NormalizePackageName(name))
+		normalizedName := NormalizePackageName(name)
+		_, isFound := encountered[normalizedName]
+
+		if !isFound {
```

Comment:
nit: in Go you can do

```go
if _, exists := encountered[normalizedName]; !exists {
  ...
}
```

Reply:
Thanks Oliver!  I thought I remembered seeing something like that before, but I couldn't seem to find it last night.  I'll get it updated a bit later today.